pymeasure/instruments/tektronix/afg3152c.py

```diff
@@ -23,66 +23,103 @@
 #
 from pymeasure.instruments import Instrument, RangeException
 from pymeasure.instruments.validators import strict_range, strict_discrete_set
-from math import sqrt,log10
 
 class Channel(object):
-
-        SOURCE_VALUES= [1,2]
+        SHAPES={
```

Comment:
Please follow PEP8 style (putting whitespaces around operators). Note that this does not apply to keyword arguments. https://www.python.org/dev/peps/pep-0008/#other-recommendations